WhisperAPI/WhisperAPI/Controllers/SuggestionsController.cs

```diff
@@ -40,5 +40,25 @@ public IActionResult GetSuggestions([FromBody] SearchQuery searchQuery)
 
             return this.Ok(suggestion);
         }
+
+        [Route("/Whisper/[Controller]/click")]
```

Comment:
Click is not really a valid rest endpoint, you incorporated UI action as a rest resource